components/RefinementList/RefinementList.js

```diff
@@ -35,7 +35,9 @@ class RefinementList extends React.Component {
       [this.props.cssClasses.active]: facetValue.isRefined
     });
 
-    let key = facetValue[this.props.facetNameKey] + '/' + facetValue.isRefined + '/' + facetValue.count;
+    let key = facetValue[this.props.facetNameKey] + '/' + facetValue.isRefined;
+    if (facetValue.count !== undefined)
+      key += '/' + facetValue.count;
```

Comment:
I think is refined is good since it's mandatory in both numericRefinementList and refinementList